chromium/manifest.json

```diff
@@ -9,35 +9,35 @@
             "background.js",
             "incognito-cache-clearing.js"
         ]
-    }, 
+    },
     "browser_action": {
         "default_icon": {
-            "38": "icon38.png"
-        }, 
-        "default_popup": "popup.html", 
+          "38": "icons/icon-active-38.png"
+        },
+        "default_popup": "popup.html",
         "default_title": "__MSG_about_ext_name__"
-    }, 
-    "default_locale": "en", 
-    "description": "__MSG_about_ext_description__", 
-    "devtools_page": "devtools.html", 
-    "homepage_url": "https://www.eff.org/https-everywhere", 
+    },
+    "default_locale": "en",
+    "description": "__MSG_about_ext_description__",
+    "devtools_page": "devtools.html",
+    "homepage_url": "https://www.eff.org/https-everywhere",
     "icons": {
-        "128": "icon128.png", 
-        "16": "icon16.png", 
-        "48": "icon48.png"
-    }, 
-    "incognito": "spanning",
-    "manifest_version": 2, 
+        "48": "icons/icon-active-48.png",
+        "128": "icons/icon-active-128.png"
+    },
+    "incognito": "split",
```

Comment:
Any reason you went back to split incognito? (I'm guessing this was a merge issue)